app/models/shipit/pull_request.rb

```diff
@@ -36,6 +38,8 @@ def required_statuses
     belongs_to :merge_requested_by, class_name: 'Shipit::User'
     has_one :merge_commit, class_name: 'Shipit::Commit'
 
+    deferred_touch stack: :updated_at
```

Comment:
We use `stack.updated_at` to populate the Stack#show `Last-Modified` header. Anything that might change the content of that page (many things), must touch the stack to clear the cache.

`deferred_touch` is like standard `touch :stack` (akak touch parent), but it's debounced.